### PR TITLE
Fix layout printing support on iOS

### PR DESCRIPTION
**Pull request by nirvn**
Let iOS users go wild printing their maps in the field too.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 632103a1d992655cb8bdbe249a3d3dd0e98da69e: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3678-d475cc3e8bc2768814511015065980eae9bb8f12-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3678-d475cc3e8bc2768814511015065980eae9bb8f12-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3678-d475cc3e8bc2768814511015065980eae9bb8f12-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3678-d475cc3e8bc2768814511015065980eae9bb8f12-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3678-d475cc3e8bc2768814511015065980eae9bb8f12-arm64-android.apk)
- [all access arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3678-d475cc3e8bc2768814511015065980eae9bb8f12-arm-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3678-d475cc3e8bc2768814511015065980eae9bb8f12-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3678-d475cc3e8bc2768814511015065980eae9bb8f12-x86-android.apk)
</details>

**Comment by nirvn** [comment]
@3nids , it'd be great if you could review this.

File: src/core/platforms/ios/iosplatformutilities.mm [hunk 1]
```diff
@@ -35,6 +36,28 @@
 
 IosPlatformUtilities::IosPlatformUtilities() : PlatformUtilities() {}
 
+@interface DocViewController
```
Comment (3nids) [inline src/core/platforms/ios/iosplatformutilities.mm:39]: this part of the code was in a separate header in the demo code. Any reason behind this?

**Review by 3nids — APPROVED** [review]
I have tested on device and works as expected, excellent :)

Only minor remark would be to move the DocViewController code to separate files (h +mm) to have the code in its own place and stay close to the demo code.
But I don't mind.

Nice work!

File: src/core/platforms/ios/iosplatformutilities.mm (hunk 1 above)
Comment (nirvn) [reply src/core/platforms/ios/iosplatformutilities.mm:39]: @3nids , not really other than I thought it was nifty to keep it within platform utilities as it's ever only used there. Got mild feelings about it, can easily be changed if you have stronger feelings :)

Thanks for the review.

Comment (3nids) [reply src/core/platforms/ios/iosplatformutilities.mm:39]: nah, keep it simple and move forward 👍

**Pull request merged**